lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -13,6 +13,37 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
 
   @relevant_effects ~w[shuttle suspension station_closure delay]a
 
+  @gl_eastbound_split_stops [
```

Comment:
I had this happen to me too. Really not sure why. The PR was small enough that I just remade the branch and that fixed it. Not a viable solution here though 😬